src/Frontend/Modules/FormBuilder/Widgets/Form.php

```diff
@@ -317,7 +317,7 @@ private function parse()
             $this->tpl->assign('fields', $this->fieldsHTML);
 
             // parse form
-            $this->frm->parse($this->tpl);
+            $this->frm->render($this->tpl);
```

Comment:
Similar to src/Frontend/Core/Engine/Base/Widget.php
Follow the cookie trail extension tpl comes from the object class.

In short  Spoon template files where compiled piece by piece with new instance of template. 
But twig doesn't work like that, so I get my Template Class already instantiated in the Object class.